src/components/Facility/FacilityUsers.tsx

```diff
@@ -94,7 +103,7 @@ export default function FacilityUsers(props: { facilityId: number }) {
       <UserListView
         users={userListData?.results ?? []}
         onSearch={(username) => updateQuery({ username })}
-        searchValue={qParams.username}
+        searchValue={username || ""}
```

Comment:
let's not over complicate this, you can just use qParams.username, it will handle it.